charts/hono/Chart.yaml

```diff
@@ -46,7 +46,7 @@ dependencies:
   condition: "grafana.enabled"
 - name: "mongodb"
   repository: "https://charts.bitnami.com/bitnami"
-  version: "^13.x"
+  version: "~13.16.x"
```

Comment:
is this related to the issue of overriding the release name?
If not, I would prefer to do any necessary changes to the Mongo DB version in a separate issue/PR ...

Reply:
It is to fix this issue https://github.com/eclipse/packages/pull/469#issuecomment-1696238963

Reply:
Created separate PR #491 for that.